src/Components/Form/Textarea/Textarea.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useCallback, useEffect } from "react";
+
+import "./Textarea.scss";
+
+const Textarea = ({
```

Comment:
Could you also add a small indicator where you can see how many characters you have already typed in?

Reply:
<img width="937" alt="image" src="https://user-images.githubusercontent.com/16488024/117563078-bf441500-b0a3-11eb-8198-3e0e10cab757.png">

Reply:
I don't think we need that much rows for 250. A maximum of 3 rows should be fine
![2021-05-09_10-24](https://user-images.githubusercontent.com/54618409/117565305-caea0880-b0b0-11eb-831d-1f8285b0715d.png)

Reply:
It depends on the window size and the usage of new lines.

Reply:
Is the resize icon in the bottom right corner not showing on windows?